docs/en/siem/siem-ui.asciidoc

```diff
@@ -65,29 +65,164 @@ versions, machine type, and so forth.
+add location information to the event. For example, you can use the Console in
+{kib} to create the following pipeline:
++
+--
+[source,json]
+----
+PUT _ingest/pipeline/geoip-info
+{
+  "description": "Add geoip info",
+  "processors": [
+    {
+      "geoip": {
+        "field": "client.ip",
+        "target_field": "client.geo",
+        "ignore_missing": true
+      }
+    },
+    {
+      "geoip": {
+        "field": "source.ip",
+        "target_field": "source.geo",
+        "ignore_missing": true
+      }
+    },
+    {
+      "geoip": {
+        "field": "destination.ip",
+        "target_field": "destination.geo",
+        "ignore_missing": true
+      }
+    },
+    {
+      "geoip": {
+        "field": "server.ip",
+        "target_field": "server.geo",
+        "ignore_missing": true
+      }
+    },
+    {
+      "geoip": {
+        "field": "host.ip",
+        "target_field": "host.geo",
+        "ignore_missing": true
+      }
+    }
+  ]
+}
+----
+//CONSOLE
+--
++
+In this example, the pipeline ID is `geoip-info`. `field` specifies the field
+that contains the IP address to use for the geographical lookup, and
+`target_field` is the field that will hold the geographical information.
+`"ignore_missing": true` configures the pipeline to continue processing when
+it encounters an event that doesn't have the specified field.
+
+. If you want to add your network’s internal IP addresses to the map,
+define geo location fields under the `processors` tag in the Beats configuration files
+on your hosts:
++
+[source,yml]
+----------------------------------
+  processors:
+   - add_host_metadata:
+   - add_cloud_metadata: ~
+   - add_fields:
+       when.network.source.ip: <private/IP address> <1>
+       fields:
+         source.geo.location:
+           lat: <latitude coordinate>
+           lon: <longitude coordinate>
+       target: ''
+   - add_fields:
+       when.network.destination.ip: <private/IP address>
+       fields:
+         destination.geo.location:
+           lat: <latitude coordinate>
+           lon: <longitude coordinate>
+       target: ''
+----------------------------------
+<1> For the IP address, you can use either `private` or CIDR notation.
++
+TIP: You can also enrich your data with other
+{packetbeat-ref}/add-host-metadata.html[host fields].
+
+. Add the pipeline to the `output.elasticsearch` tag:
++
+[source,yml]
+----------------------------------
+  output.elasticsearch:
+    hosts: ["localhost:9200"]
+    pipeline: geoip-info <1>
+----------------------------------
+<1> The value of this field must be the same as the ingest pipeline name in
+<<geo_pipeleine, step 1>> (`geoip-info` in this example).
+
+[float]
+[[map-actions]]
+==== Interact with the map
+Hover over source and destination points to see more information, such as hostnames
+and IP addresses. To drill down, click a point and use the plus sign to add a field
+to the query bar. You can also click a hostname to jump to the SIEM Host page, or
```

Comment:
Sorry for the last-minute comment, but just thought of this -- you can only click on the hostname & source.ip/destination.ip if you set up a [URL field formatter](https://www.elastic.co/guide/en/kibana/current/field-formatters-string.html) for these fields in the relevant indices.

Here's the necessary config for each:

Field: `host.name`
URL Template: `[BASE_URL]/app/siem#/network/ip/{{value}}`
Label Template: `{{value}}`

<img src="https://user-images.githubusercontent.com/2946766/65060419-0b719d00-d935-11e9-893f-b004a856b54a.png" width=200>


Field: `source.ip` / `destination.ip`
URL Template: `[BASE_URL]/app/siem#/network/ip/{{value}}`
Label Template: `{{value}}`

<img src="https://user-images.githubusercontent.com/2946766/65060204-addd5080-d934-11e9-8896-8f4eaabbdc0f.png" width=200>

Reply:
No problem. I added the steps, but I'm debating putting all map configuration steps on a separate page. What do you think?
I'll also consult with @karenzone

Reply:
@spong please don't review yet - I'm going to put the configuration instructions on a separate page. I'll ping you when it's ready - probably tomorrow.

Reply:
Sounds good! I think that's the right move. Then that keeps the main guide page specific to each overview item without going into the weeds of configuration.

Reply:
@spong 
Updated with map configuration on its own page. Thanks,

Reply:
Much better flow with the configuration on its own page -- very nice! Thanks @benskelker! 🙂